bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/shape-draw-listener/component.jsx

```diff
@@ -151,14 +151,21 @@ export default class ShapeDrawListener extends Component {
     // Sometimes when you Alt+Tab while drawing it can happen that your mouse is up,
     // but the browser didn't catch it. So check it here.
     if (this.isDrawing) {
+      if (event.button === 2) {
+        this.props.actions.undoAnnotation(this.props.whiteboardId);
+        this.isDrawing = false;
```

Comment:
It prevents the button for discarding an annotation working weird, right after discarding the current drawing using mouse right button.